lib/linter.coffee

```diff
@@ -0,0 +1,147 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+fs = require 'fs'
+fse = require 'fs-extra'
+path = require 'path'
+temp = require 'temp'
+XRegExp = require('xregexp').XRegExp
```

Comment:
Wow, you're fast. :)

My biggest reason for not using the helper module for parsing is that is not able to take into account indenting and ends up underlining all the leading whitespace.  My parsing [uses the textEditor object to get the indentation level](https://github.com/AtomLinter/linter-haml/pull/11/files#diff-8bbae72b6288a0263998cd68a1086738R113) so it can be taken into account (a trick I picked up from [linter-coffeelint](https://github.com/AtomLinter/linter-coffeelint/blob/master/lib/plus-linter.coffee#L29)).  Is there some way to have the best of both worlds?